### DISCUSSION TITLE
Display maximum combo as a percentage and not a number

### DISCUSSION THREAD

**Discussion by Natelytle** [ppy/osu#21426] [Ideas]
Because the max combo of a map is no longer consistent due to classic mod, it would be better UX wise to display the percentage of the max combo achieved instead of a raw number. In the below screenshot the top play has a higher percentage of the map combo'd, but the combo number is lower because the max combo of the map is 410 without classic mod and 559 with classic mod.
![image](https://user-images.githubusercontent.com/92956514/204074830-936fb52b-38f7-45a9-8681-6c17b62f99cb.png)

**Comment by peppy** [comment]
I disagree with percentage, but it would be good to show what the actual max is next to the number or somewhere else. Or change the display to show "combo breaks" instead.
